src/components/crossDevice/GenericError/index.js

```diff
@@ -3,17 +3,20 @@ import { h, Component } from 'preact'
 import Title from '../../Title'
 import theme from '../../Theme/style.css'
 import style from './style.css'
+import errors from '../../strings/errors'
+import { lowerCase } from '../../utils/string'
 import { sendScreen } from '../../../Tracker'
 import { localised } from '../../../locales'
 
 class GenericError extends Component {
   componentDidMount() {
-    sendScreen(['generic_client_error'])
+    sendScreen([`${lowerCase(this.props.error.name)}`])
```

Comment:
why does it need to be converted to lower case?

Reply:
is this for tracking or something? o.O

Reply:
Yes, that's for tracking. It doesn't have to but all the other events sent are lower case

Reply:
alright, no issues then. point to consider in the future to move that lowercase to inside the `sendScreen` function, or maybe a more lower level tracking function.

Reply:
+1 on moving the `lowercase` logic to an entrypoint to the tracking function. No sense littering `lowercase(...)` everywhere, when it could be written just once 🙂